keyboards/willow/willow64/keymaps/buildtest/keymap.c

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2021 Hanachi
```

Comment:
What's the purpose of this keymap?

Reply:
This is used to check the lighting of the LED when soldering.